contrib/babelfishpg_tsql/src/catalog.c

```diff
@@ -3063,6 +3063,9 @@ add_entry_to_bbf_schema_perms(const char *schema_name,
 
 	/* Close bbf_authid_user_ext, but keep lock till commit */
 	table_close(bbf_schema_rel, RowExclusiveLock);
+
+	/* make sure later steps can see the entry added here */
+	CommandCounterIncrement();
```

Comment:
For commands like `GRANT SELECT, INSERT on t1 to u1`, SELECT privilege will add an entry to the catalog and the second iteration of INSERT privilege should be able to see the first entry in the catalog so that it updates the permission column.